EngineLayer/CommonParameters.cs

```diff
@@ -104,15 +104,5 @@ public CommonParameters()
         public bool TrimMsMsPeaks { get; private set; }
         public bool UseDeltaScore { get; private set; }
         public bool CalculateEValue { get; private set; }
-
-        public CommonParameters Clone()
```

Comment:
plz put this back

Reply:
Is there a reason? I didn't see it being used for anything.

Reply:
it's not used right now, but I'd like to use it to duplicate a task in the GUI